docs/runner.rst

```diff
@@ -64,7 +64,12 @@ Invocation
 
 Usage::
 
-    waitress-serve [OPTS] MODULE:OBJECT
+    waitress-serve [OPTS] --app=MODULE:OBJECT
+
+Required options:
+
+``--app``
+    Specify WSGI application to run.
```

Comment:
You'll break the existing external interface this way.

You should be able to accept the application using both the `--app=` flag _and_ by passing it as a positional argument.

Reply:
I will - but first, I need a general agreement of how everything implemented, and is argparse is "enough" to solve problem.

On the "positional app argument" - it is not clear now, should this argument be "still first one", and for passing app eg. last one should use --app form. Or it app should be accept eg. first or last - but this will complicate parsing.

Reply:
On "argparse" side, right now there are some cases where it is hard to achieve desired results, eg. mutually exclusive groups of options. For now it can be solved as external "validation" step after parsing, and it will suffice, I think.